### PR TITLE
Update Ansible to 2.1.2.0.

### PR DESCRIPTION
**Pull request by simono**
Use Ruby 1.9 hash syntax.

### CODE REVIEW

File: Formula/ansible.rb [hunk 1]
```diff
@@ -3,10 +3,10 @@ class Ansible < Formula
 
   desc "Automate deployment, configuration, and upgrading"
   homepage "https://www.ansible.com/"
-  url "https://releases.ansible.com/ansible/ansible-2.1.1.0.tar.gz"
-  sha256 "61e739c123923ba90169f42c54e5f51df759ed40b4e332a7160d7db963d5678b"
+  url "https://releases.ansible.com/ansible/ansible-2.1.2.0.tar.gz"
+  sha256 "9c37a7bd397c05ab8ca3fcc49417649ea49b9133d4cd9500408235617d1621eb"
 
-  head "https://github.com/ansible/ansible.git", :branch => "devel"
+  head "https://github.com/ansible/ansible.git", branch: "devel"
```
Comment (DomT4) [inline Formula/ansible.rb]: Leave this how it was for now please.

**Comment by DomT4** [comment]
Do any of the million resources need updates as well?

File: Formula/ansible.rb (hunk 1 above)
Comment (simono) [inline Formula/ansible.rb]: Done.

**Comment by simono** [comment]
I'm not sure about updates to the resources. Is there an easy way to find out?

**Comment by MikeMcQuaid** [comment]
@DomT4 I think we're fine as-is unless we know there's an issue here.

**Comment by DomT4** [comment]
Ansible has proven to be awfully fond of silent breakage before on the resource front, but I can't say I care much. It's just a nightmare having this many resources in a single formula. If it breaks, I'm sure we'll find out soon enough though, so I'm fine with shipping it and seeing how we go.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `ansible 2.1.2.0.`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**